frontend/lib/controllers/api_controller.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:dio/dio.dart';
+import 'package:flutter_dotenv/flutter_dotenv.dart';
+import 'package:frontend/sdk/user.dart';
+import 'package:frontend/sdk/workflow.dart';
+import 'package:get/get.dart';
+
+class ApiController extends GetxController {
+  static ApiController instance = Get.find();
+
+  User? user;
+  final Dio dio = Dio(
+    BaseOptions(
+      baseUrl: dotenv.env['API_URL'] ?? '',
```

Comment:
There should be a default api URL, `http://localhost:8080` for instance

Reply:
I will change that as you will, but, to me, the shouldn't work if a proper .env hasn't been set.